t/ui/18-tests-details.t

```diff
@@ -47,6 +47,12 @@ unless ($driver) {
     exit(0);
 }
 
+sub disable_bootstrap_fade_animation {
```

Comment:
if it wasn't required, he wouldn't call the function after all. Nonsense comments are just noise